Ash.FrontEnd/src/global/style.less

```diff
@@ -0,0 +1,20 @@
+.ash-section {
+  margin: 10px;
+  border-radius: 10px;
+  background-color: #333;
+
+  div {
+    padding: 20px 20px 10px;
+
+    h1 {
+      margin: 0;
+      font-size: 36px;
+      font-weight: 380;

```

Comment:
I didn't realize you could specify values between the 100's divisions lol!

Reply:
Lol yeah, though think my zoom was just being weird, I should probably change to 400